packages/datetime/src/common/timezoneUtils.ts

```diff
@@ -69,7 +68,7 @@ export function getDateObjectFromIsoString(
     if (value === undefined) {
         return undefined;
     }
-    if (value === null || isEmpty(value)) {
+    if (value === null || value === "") {
```

Comment:
```suggestion
    if (value === null || value === "" || typeof value !== "string") {
```

Can you add a `typeof` check here to be extra defensive in the event a non-TypeScript consumer is passing in an incorrect value here (such as a number) and expecting to get `null` returned?

`isEmpty` evaluates to `true` on numbers, so I'd like to preserve that behavior to avoid an accidental break.

Reply:
Seems reasonable. Interesting that here lodash.isEmpty very much *doesn't* do what it [says on the tin](https://lodash.com/docs/4.17.15#isEmpty):

>Checks if `value` is an empty object, collection, map, or set.